mentoring/mrq.py

```diff
@@ -94,5 +100,13 @@ def submit(self, submissions):
             'score': float(score) / len(results),
         }
 
+        return result
+
+    def submit(self, submissions):
```

Comment:
@Kelketek same nit here as everywhere :) Can (probably) be placed before `get_results` to reduce diff.